rocketpy/simulation/df2flight.py

```diff
@@ -0,0 +1,208 @@
+"""Starts with a pandas DataFrame containing the rocket's collected flight data
+and build a rocketpy.Flight object from it.
+"""
+
+from dataclasses import dataclass
+from functools import cached_property, wraps
+
+import numpy as np
+
+from rocketpy.mathutils import Function
+from rocketpy.tools import import_optional_dependency
+
+pandas = import_optional_dependency("pandas")
+pd = pandas
+
+
+def df2property(
```

Comment:
If this decorator only works inside the `DataFrameToFlight` class, wouldn't it be better to move it there?

See https://www.geeksforgeeks.org/creating-decorator-inside-a-class-in-python/

Also, consider other names such as `df_2_function_property` or similar. I feel we are missing the Function class in the name, since it is core part of this decorator.

Reply:
Yes, it would be a better idea.

But the decorator no longer exists.